Shared/Samples/Edit feature attachments/EditFeatureAttachmentsView.swift

```diff
@@ -0,0 +1,228 @@
+                            let attachment = model.attachments[index]
+                            AttachmentView(attachment: attachment, onDelete: { attachment in
+                                Task {
+                                    do {
+                                        try await model.deleteAttachment(attachment)
+                                    } catch {
+                                        self.error = error
+                                    }
+                                }
+                            })
+                        }
+                    }
+                }
+                Section {
+                    AddAttachmentView(onAdd: {
+                        Task {
+                            do {
+                                guard let pngData = UIImage(
+                                    named: "PinBlueStar"
+                                )?.pngData() else {
+                                    return
+                                }
+                                try await model.addAttachment(
+                                    named: "Attachment",
+                                    type: "png",
+                                    dataElement: pngData
+                                )
+                            } catch {
+                                self.error = error
+                            }
+                        }
+                    })
+                }
+            }
+            .navigationTitle("Attachments")
+            .navigationBarTitleDisplayMode(.inline)
+            .toolbar {
+                ToolbarItem(placement: .confirmationAction) {
+                    Button("Done") {
+                        dismiss()
+                    }
+                }
+            }
+            .errorAlert(presentingError: $error)
+        }
+    }
+}
+
+private extension EditFeatureAttachmentsView {
+    // MARK: - CalloutView
+    
+    struct CalloutView: View {
+        /// The data model for the sample.
+        @ObservedObject var model: Model
+        
+        var body: some View {
+            VStack(alignment: .leading, spacing: 3) {
+                Text(model.calloutText)
+                    .font(.callout)
+                    .multilineTextAlignment(.leading)
+                Text(model.calloutDetailText)
+                    .font(.footnote)
+                    .multilineTextAlignment(.leading)
+            }
+        }
+    }
+}
+
+private extension EditFeatureAttachmentsView {
+    // MARK: - AttachmentView
+    
+    struct AttachmentView: View {
+        // The attachment that is being displayed.
+        let attachment: Attachment
+        // The closure called when the delete button is tapped.
+        let onDelete: ((Attachment) -> Void)
+        // The image in the attachment.
+        @State private var image: Image?
+        
+        var body: some View {
+            HStack {
+                Text("\(attachment.name)")
```

Comment:
```suggestion
                Text(attachment.name)
```
`attachment.name` is already a `String` so string interpolation is not needed here.